src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -146,8 +146,10 @@ interface MongoLayerMapping {
   lookaheadMaxSearchDistance: number
 }
 
-export interface MongoMedia {
+export interface MongoMedia extends MongoId {
+  id: string
```

Comment:
It shouldn't both extends `MongoId` and have an `id` attribute

Reply:
It no longer does!